.github/workflows/e2e.yml

```diff
@@ -35,11 +35,11 @@ jobs:
       # resolution options, using npm release or a gh ref:
       #
       # option #1, set the version of soroban-js-client based on a npm release version
-      SYSTEM_TEST_JS_SOROBAN_CLIENT_NPM_VERSION: 
+      SYSTEM_TEST_JS_SOROBAN_CLIENT_NPM_VERSION: 1.0.0-beta.0
       # option #2, set the version of soroban-js-client used as a ref to a gh repo
       # if this value is present it takes precedence over any SYSTEM_TEST_JS_SOROBAN_CLIENT_NPM_VERSION
-      SYSTEM_TEST_JS_SOROBAN_CLIENT_GH_REPO: stellar/js-soroban-client
-      SYSTEM_TEST_JS_SOROBAN_CLIENT_GH_REF: refs/pull/135/head
+      # SYSTEM_TEST_JS_SOROBAN_CLIENT_GH_REPO: stellar/js-soroban-client
+      # SYSTEM_TEST_JS_SOROBAN_CLIENT_GH_REF: refs/pull/135/head
```

Comment:
just a question for myself: can this be empty or does it need to be commented?

Reply:
good question, it was kinda ambiguous, so, I updated this to be more direct, empty values are the way, don't comment out- https://github.com/stellar/soroban-tools/pull/952/commits/c6c26e515c3eafab505d058ea2a14b0abf4d0197